### ISSUE TITLE
Share Memory Options ($100)

### ISSUE THREAD

**Issue by josancamon19** [BasedHardware/Friend#382]
**Is your feature request related to a problem? Please describe.**
On memory detail, right next to the 3 dots, there should be a share option, that also opens a bottom sheet.
This share option should have:

- Export Audio
- Export Transcript
- Export Summary
- Full export (or different name) which should include everything to be exported, transcript, summary, plugins.

![IMG_4704](https://github.com/BasedHardware/Friend/assets/18078879/89cbcfe8-24ff-4ca0-b68f-1244e48c1bc0)

FOR EVERY SHARING MATERIAL, AT THE TOP IT SHOULD SAY "GENERATED BY FRIEND, basedhardware.com"

**Comment by josancamon19** [comment]
The bottom sheet 3 dots option that says "Share Memory" should be removed

**Comment by josancamon19** [comment]
Should Include export in plain text, pdf, and markdown, so it can be saved in obsidian or anywhere else.

https://feedback.limitless.ai/b/kvj1nlve/feature-ideas/export-function-to-markdown

**Comment by Wolfof420Street** [comment]
Hi could you assign this to me please @josancamon19

**Comment by josancamon19** [comment]
Assigned to @Wolfof420Street for the next 2 days

**Comment by Wolfof420Street** [comment]
@josancamon19 is [this](https://github.com/Wolfof420Street/Friend/tree/382) implementation on the right track? tried messaging you n disscord for clarification

**Comment by josancamon19** [comment]
@Wolfof420Street discord blocks messages sometimes, please create a Draft PR to look at more details, and any questions you might have, please post them here.

**Comment by Wolfof420Street** [comment]
@josancamon19 I have created a draft PR. was wondering if I should remove the Share options and include all in the image above?

**Comment by Wolfof420Street** [comment]
@josancamon19 any feedback on this?

**Comment by josancamon19** [comment]
Please share transfer details @Wolfof420Street on discord to @josancamon19
TLDR, task merged, not fully made, had to refactor many things, and comment many useless things, so I will transfer accordingly.

**Comment by Wolfof420Street** [comment]
@josancamon19 shared the details on discord.  My username is wolfof420Street69

**Comment by Wolfof420Street** [comment]
@josancamon19 shared the details on discord. My username is wolfof420Street69

**Comment by Wolfof420Street** [comment]
hello @josancamon19
